test/scales/colorScaleTests.ts

```diff
@@ -0,0 +1,129 @@
+///<reference path="../testReference.ts" />
+
+describe("Scales", () => {
+  describe("Color Scale", () => {
+
+    describe("Basic Usage", () => {
+      let defaultColors = [
+        "#5279c7", "#fd373e", "#63c261", "#fad419", "#2c2b6f",
+        "#ff7939", "#db2e65", "#99ce50", "#962565", "#06cccc"
+      ];
+
+      it("default colors are generated", () => {
+        let scale = new Plottable.Scales.Color();
+        assert.deepEqual(scale.range(), defaultColors, "The color scale uses the default colors by default");
+      });
+
+      it("uses altered colors if size of domain exceeds size of range", () => {
+        let scale = new Plottable.Scales.Color();
+        let colorRange = ["#5279c7", "#fd373e"];
+        scale.range(colorRange);
+        scale.domain(["a", "b", "c"]);
+        assert.strictEqual(scale.scale("a"), colorRange[0], "first color is used");
+        assert.strictEqual(scale.scale("b"), colorRange[1], "second color is used");
+        assert.notStrictEqual(scale.scale("c"), colorRange[0], "first color is slightly modified and used");
```

Comment:
can you add an assertion that scale.scale("c") is not equal to colorRange[1] as well?
also, the assertion message is confusing as you only tested those 2 colors are not equal, you didn't test that one is a slight modification of another